### PR TITLE
BIP142 - Typos and grammar

### PR DESCRIPTION
**Pull request by JornC**

### CODE REVIEW

File: bip-0142.mediawiki [hunk 1]
```diff
@@ -126,7 +126,7 @@ when encoded as a P2PKH template, would become:
   
     DUP HASH160 <010966776006953D5567439E5E39F86A0D273BEE> EQUALVERIFY CHECKSIG
 
-And the corresponding version 1 Bitcoin address is
+With the corresponding version 1 Bitcoin address being:
```
Comment (JornC) [inline bip-0142.mediawiki]: Technically this is version 0 - not sure what the common nomenclature is here.

File: bip-0142.mediawiki [hunk 2]
```diff
@@ -134,7 +134,7 @@ When the same public key is encoded as P2WPKH, the scriptPubKey becomes:
   
     OP_0 <010966776006953D5567439E5E39F86A0D273BEE>
 
-Using 0x06 as witness version, followed 0x00 as witness version, and a 0x00 padding, the equivalent P2WPKH address is:
+Using 0x06 as witness version, followed by 0x00 as witness version, and a 0x00 padding, the equivalent P2WPKH address is:
```
Comment (jonasschnelli) [inline bip-0142.mediawiki]: You might want to include https://github.com/bitcoin/bips/pull/290 here.
It should be `Using 0x06 as address version, followed by 0x00 as witness program version[...]`

Comment (JornC) [inline bip-0142.mediawiki]: Rebased

File: bip-0142.mediawiki (hunk 1 above)
Comment (jl2012) [inline bip-0142.mediawiki]: It is called version 1 address on the Bitcoin wiki: https://en.bitcoin.it/wiki/Technical_background_of_version_1_Bitcoin_addresses

**Comment by jl2012** [comment]
ACK and thanks!

**Comment by harding** [comment]
LGTM.

**Pull request merged**